module/merintr/inventry.c

```diff
@@ -1062,6 +1076,65 @@ Bool InventoryDropCurrentItem(room_contents_node *container)
    return True;
 }
 /************************************************************************/
+/*
+ * InventoryMoveCurrentItem:  Move the selected item with the inventory cursor 
+ *   to the given (x, y) coordinates, if any.  Return True iff item moved.
```

Comment:
Clarify first comment.. item is moved to *the item at* the given coordinates (it isn't moved to those coordinates, exactly).

Reply:
Refactored to include it is the item at the coordinates that we are moving the item to